stable/kommander/requirements.yaml

```diff
@@ -1,14 +1,10 @@
 dependencies:
-- name: kubefed
```

Comment:
So kubefed is being removed? Where is this applied now? and wouldnt some crds need to be removed now? what are the assumptions here?

Reply:
That stuff has been moved to the kcl repohttps://github.com/mesosphere/kommander-cluster-lifecycle/pull/313